examples/MatLabAgent_v2/config

```diff
@@ -0,0 +1,7 @@
+{
+  # VOLTTRON config files are JSON with support for python style comments.
+  "script_names": ["testScript2.py"], 
+  "script_args": [["20"]],
+  "topics_to_agents": ["matlab/to_matlab/1"], 
```

Comment:
Update example to have the correct config. topics_to_matlab and topics_to_volttron